packages/captp/test/prepare-test-env-ava.js

```diff
@@ -1,5 +1,6 @@
 // @ts-check
-import './prepare-test-env.js';
+import '@agoric/install-ses/get-lockdown.js';
```

Comment:
I think this line is redundant with the next, since `lockdown-debug.js` imports this as well?

Reply:
Likewise elsewhere.

Reply:
Fixed.

Reply:
Fixed.